physionet-django/console/forms.py

```diff
@@ -934,6 +941,43 @@ def clean(self):
             raise forms.ValidationError('If you reject, you must explain why.')
 
 
+class LicenseForm(forms.ModelForm):
+    class Meta:
+        model = License
+        fields = (
+            'name',
+            'version',
+            'slug',
+            'is_active',
+            'html_content',
+            'home_page',
+            'access_policy',
+            'project_types',
+        )
+        labels = {'html_content': 'Content'}
+
+
+class DUAForm(forms.ModelForm):
+    class Meta:
+        model = DUA
+        fields = (
+            'name',
+            'version',
+            'slug',
+            'is_active',
+            'html_content',
+            'access_template',
+            'access_policy',
+            'project_types',
+        )
+        labels = {'html_content': 'Content'}
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.fields['access_policy'].choices = AccessPolicy.choices(gte_value=AccessPolicy.CREDENTIALED)
```

Comment:
Restricted datasets require a DUA, so I think this should be `gte_value=AccessPolicy.RESTRICTED`

Reply:
Updated in https://github.com/MIT-LCP/physionet-build/pull/1519/commits/7a12b0d12d36b86607efb9748db5cc9608089c10